lib/class-wp-theme-json-gutenberg.php

```diff
@@ -1284,21 +1284,68 @@ protected function process_blocks_custom_css( $css, $selector ) {
 	 * @return string The global styles custom CSS.
 	 */
 	public function get_custom_css() {
```

Comment:
Same as below: can we deprecate/remove `get_custom_css`, `get_base_custom_css`, and `get_block_custom_css_nodes`, `get_block_custom_css`, in favor of the generic functions? IMO, it's too much code for a single style property that should behave like any other (and be part of the general algorithm).

Reply:
That's a good point! It would be better if we didn't need as many functions, but I don't know why it was done this way in the first place. I'll look into it as a follow-up, thanks for reviewing!